.cirrus.yml

```diff
@@ -93,7 +93,6 @@ gating_task:
         CIRRUS_WORKING_DIR: "/usr/src/libpod"
         GOPATH: "/go"
         GOSRC: "/go/src/github.com/containers/libpod"
-        EPOCH_TEST_COMMIT: "${CIRRUS_BASE_SHA}"
```

Comment:
This now leaves a dangling instance of `EPOCH_TEST_COMMIT` in a comment in this file; that will confuse future maintainers

Reply:
don't worry, both past and future maintainers are already confused.  It's a job requirement :smile: 

joking aside, I think #5039 will make the situation slightly better.

Reply:
better meaning: I added some comments